dashboard/components/Input.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { bool, func, string } from 'prop-types'
+
+import './Input.css'
+
+const Input = ({
+  label,
+  name,
+  type,
+  required,
+  value,
+  error,
+  onChange,
+  placeholder,
+  autoFocus
+}) =>
+  <div className='lk-input'>
+    <label className='lk-input__label' htmlFor={name}>
+      {label}
+      {required ? '*' : null}
+    </label>
+    <input
+      id={name}
+      className='lk-input__input'
+      value={value}
+      onChange={e => onChange(name, e.target.value)}
```

Comment:
[Minor] Add function props to d defaultProps as empty functions to avoid exceptions

```javascript
defaultProps = {
  // ...
  onChange: () => {},
  // ...
}
```